src/commonMethods/commonFunctions.js

```diff
@@ -79,16 +173,318 @@ export const calcAvgFPS = function() {
   average = average.toFixed(2)
   this.$data.write('average', average)
 }
+
+/**
+ * This function is used to get screenshot
+ * @returns {Promise<void>}
+ */
+export const screenshot = async function() {
+  let url = `http://${constants.host}:80/Service/Snapshot/Capture?${moment().valueOf()}`
+  // create a new promise inside of the async function
+  let bufferData = new Promise((resolve, reject) => {
+    _http
```

Comment:
Is there a reason for using the http library here (over the built in this.$http helper, which acts a facade for Axios library)?